ldpd/ldpd.c

```diff
@@ -277,9 +277,6 @@ main(int argc, char *argv[])
 		exit(1);
 	}
 
-	openzlog(ldpd_di.progname, "LDP", 0,
```

Comment:
With this change the child processes are not calling openzlog() anymore, which is a problem.